src/components/portal/portal.tsx

```diff
@@ -86,6 +86,13 @@ export class Portal {
     @Prop({ reflect: true })
     public visible = false;
 
+    /**
+     * The element that the content should be positioned relative to.
+     * Defaults to the limel-portal element.
+     */
+    @Prop()
+    public positioningSource?: HTMLElement = null;
```

Comment:
Is this a good name for this property? It's the best I've come up with so far, but any suggestions are most welcome! 👍

Reply:
I think anchor is a common term that is used for stuff like this

Reply:
Ah, yes, of course! Thank you! 🙏

Reply:
⚡